src/components/Gesture.svelte

```diff
@@ -315,25 +326,23 @@
   </div>
 {:else}
   <div class="max-w-max {isGestureNamed || hasRecordings ? 'visible' : 'invisible'}">
-    <GestureTilePart small elevated>
+    <GestureTilePart small elevated selected={isChosenGesture} onClick={selectGesture}>
```

Comment:
We can pass through `on:click={selectGesture}` to GestureTilePart by simply adding `on:click` to GestureTilePart. The `onClick` prop isn't needed.

We have onClick for the standard and icon buttons to avoid diff with upstream.

Reply:
Oh, I see: https://github.com/microbit-foundation/ml-trainer/pull/123/commits/2e57077823fa45854b6b1661b907f65d807e71f5